### PR TITLE
Fixed Angular Compile Issues

### PR DESCRIPTION
**Pull request by erichunsinger**

### CODE REVIEW

File: src/app/pages/jobs/job-modal/job-modal.component.html [hunk 1]
```diff
@@ -40,12 +40,10 @@ <h4 class='title areas'>Skills</h4>
                             </mat-chip>
                         </mat-chip-list>
                     </div>
-                    <div *ngIf='job.filled' id='filledprofile'>
-                        <h4 class='title areas'>Filled By</h4>
-                        <br>
-                        {{ job.profile.firstName }} {{ job.profile.lastName }}
+                    <div *ngIf='job.isFilled' id='filledprofile'>
```
Comment (IkenoXamos) [inline src/app/pages/jobs/job-modal/job-modal.component.html:43]: Why was this change made?

**Review by IkenoXamos — COMMENTED** [review]
I am curious about one change. Is it needed?

File: src/app/pages/jobs/job-modal/job-modal.component.html (hunk 1 above)
Comment (JWulf7) [reply src/app/pages/jobs/job-modal/job-modal.component.html:43]: id job.profile did not exist on the backend. There was no use for this particular piece of code. Pointed to a null variable, which caused compilation errors.

**Review by IkenoXamos — APPROVED** [review]
Looks good!

**Pull request merged**